common-content/en/module/tools/first-nodejs-program/index.md

```diff
@@ -0,0 +1,114 @@
++++
+title = "Writing a NodeJS program"
+headless = true
+time = 60
+facilitation = false
+emoji= "🛠️"
+hide_from_overview = true
+objectives = [
+    "Write a zero-dependencies NodeJS program",
+]
++++
+
+Below we have a small NodeJS program. It is a bit like `wc`. It counts words in a file. Specifically, it counts words which contain a hyphen (`-`) character.
```

Comment:
```suggestion
Below we have a small NodeJS program. It is a bit like `wc`. It counts words which contain a hyphen (`-`) character. Our program is called hyphenWC.
```

Reply:
Rephrased to instead have a bunch of "Our program"s rather than "It"s or naming the program artificially.